python-stdlib/logging/logging.py

```diff
@@ -1,128 +1,212 @@
 import sys
+import time
 
-CRITICAL = 50
-ERROR = 40
-WARNING = 30
-INFO = 20
-DEBUG = 10
 NOTSET = 0
+DEBUG = 10
+INFO = 20
+WARNING = 30
+ERROR = 40
+CRITICAL = 50
```

Comment:
please can you keep the original order of these definitions, just to minimise the diff so we can see exactly what changed